### ISSUE TITLE
Not able to Download Practice Tests

### ISSUE THREAD

**Issue by johnfolia** [FaisalUmair/udemy-downloader-gui#532]
Not able to Download Practice Tests

**Comment by vega7473** [comment]
Yep same here. Would be useful if we could. They are not videos but forms/templates. Maybe they could be converted into some word/doc format ?

**Comment by SillyRoeDeer** [comment]
It's the same with me.Previously, attachments could be downloaded with version 1.82, but recently, I can't, so I can only ignore the attachment and download the course directly
